support/src/main/java/org/jenkinsci/plugins/workflow/support/steps/input/ApproverAction.java

```diff
@@ -0,0 +1,25 @@
+package org.jenkinsci.plugins.workflow.support.steps.input;
+
+import hudson.model.InvisibleAction;
+import org.kohsuke.stapler.export.Exported;
+import org.kohsuke.stapler.export.ExportedBean;
+
+/**
+ * Created by valentinaarmenise on 21/01/2015.
+ */
+
+@ExportedBean
+public class ApproverAction extends InvisibleAction {
+
+    public ApproverAction(String approverName) {
+        this.approverName = approverName;
+    }
+
+    final private String approverName;
```

Comment:
Conventional name (matching similar classes in core) would be `userId`, to make it clear that it is `User.id`.